app/src/main/java/org/oppia/android/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -215,7 +216,7 @@ class ProfileChooserFragmentPresenter @Inject constructor(
         activity.startActivity(
           AdminPinActivity.createAdminPinActivityIntent(
             activity,
-            chooserViewModel.adminProfileId.internalId,
+            activity.intent.extractCurrentUserProfileId(),
```

Comment:
For all the usages in this class, isn't it supposed to be `decorateWith` instead of extract? Also, could this be a global var in this class?

Are the profile IDs in `model.profile.id` different from the one from `activity.intent.extractCurrentUserProfileId()`

Reply:
Decorate packs the value in intent while extract unpacks the same.